crud/common/utils.lua

```diff
@@ -199,6 +199,84 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
+local function map_to_table(map)
+    local tab = {}
+    for _, value in pairs(map) do
+        table.insert(tab, value)
+    end
+
+    return tab
+end
+
+local function add_nullable_fields_recursive(operations, space_format, tuple, id)
+    if id < 2 or tuple[id - 1] ~= box.NULL then
+        return operations
+    end
+
+    if space_format[id - 1].is_nullable and not operations[id - 1] then
+        operations[id - 1] = {'=', id - 1, box.NULL}
+        return add_nullable_fields_recursive(operations, space_format, tuple, id - 1)
+    end
+
+    return operations
+end
```

Comment:
Required logic if you get an error:
  - Get tuple
  - Find field in format
  - If field is in format - check are there any field before this
  - JSONpath is not in format - just skip it and don't try to parse it
  - If there is no field in tuple - just call `table.insert(updates_list, index, {'=', field_in_tuple, box.NULL})`